zh/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -17,97 +17,34 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
 
 ## 环境准备
 
-部署前，请确认软件、资源等满足如下需求：
-
-- 资源需求：CPU 2 核+、内存 4G+
-
-    > **注意：**
-    >
-    > 对于 macOS 系统，需要给 Docker 分配 2 核+ CPU 和 4G+ 内存。详情请参考 [Mac 上配置 Docker](https://docs.docker.com/docker-for-mac/#advanced)。
-
-- [Docker](https://docs.docker.com/install/)：版本 >= 17.03
-- [Helm Client](https://helm.sh/docs/intro/install/)：版本 >= 2.11.0 并且 < 2.16.4
-- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)：版本 >= 1.12，建议 1.13 或更高版本
-
-    > **注意：**
-    >
-    > 不同 kubectl 版本，输出可能略有不同。
-
-- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/)：版本 >= 0.4.0
-- [net.ipv4.ip_forward](https://linuxconfig.org/how-to-turn-on-off-ip-forwarding-in-linux) 需要被设置为 `1`
+- [docker](https://docs.docker.com/install/)：版本 >= 17.03
+- [helm](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+- [kind](https://kind.sigs.k8s.io/)
 
 ## 第 1 步：通过 kind 部署 Kubernetes 集群
 
-首先确认 Docker 进程正常运行，然后你可以通过脚本命令快速启动一个本地的 Kubernetes 集群。
-
-1. Clone 官方提供的代码：
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    git clone --depth=1 https://github.com/pingcap/tidb-operator && \
-    cd tidb-operator
-    ```
-
-2. 运行脚本，在本地创建一个 Kubernetes 集群：
-
-    {{< copyable "shell-regular" >}}
-
-    ``` shell
-    hack/kind-cluster-build.sh
-    ```
-
-    > **注意：**
-    >
-    > 通过该脚本启动的 Kubernetes 集群默认有 6 个节点，Kubernetes 版本默认为 v1.12.8，每个节点默认挂载数为 9。你可以通过启动参数去修改这些参数：
-    >
-    > {{< copyable "shell-regular" >}}
-    >
-    > ```shell
-    > hack/kind-cluster-build.sh --nodeNum 2 --k8sVersion v1.14.6 --volumeNum 3
-    > ```
+参考 `kind` [快速使用文档](https://kind.sigs.k8s.io/docs/user/quick-start) 安装
```

Comment:
people can always refer to official docs for how to install and create a cluster locally.